handlers/misttriggers/broker.go

```diff
@@ -0,0 +1,80 @@
+package misttriggers
+
+import (
+	"context"
+	"sync"
+
+	"github.com/golang/glog"
+	"golang.org/x/sync/errgroup"
+)
+
+// Broker provides an interface for allowing multiple components of the app
+// to respond to Mist trigger events without putting all of the application
+// logic into the trigger handling code itself.
+
+// There are three different cases to account for on Mist triggers; and
```

Comment:
I love this breakdown of the trigger types, really helped to clarify them in my mind and having a separate interface entry per trigger type is a nice way of handling the variances